tests/ctst/world/Zenko.ts

```diff
@@ -625,21 +745,47 @@ ${JSON.stringify(policy)}\n${err.message}\n`);
         };
     }
 
+    /**
+     * Get all saved parameters
+     * @returns {Object.<string,*>} - an object with any saved parameters
+     */
+    public getSaved(): { [key: string]: unknown } {
+        return this.saved;
+    }
```

Comment:
Souldn't this return a `Saved` type? From there, it solves other lines like this one: https://github.com/scality/Zenko/pull/1818/files#diff-371f6719b5883f728d9d3218b4ed2eaf1b7ada8faf798acf95b686a7c347c7ddR221

ALternatively, if "this.saved" already has the `Saved` type, TSC will dynamically infer the type of thie function and all the variables that you can create using it.

Reply:
The `Saved` type is an interface I created before fully understanding typing solutions I think, adding fields to the interface as they're added to the object; the `saved` object is meant to be able to take anything, so I think it actually makes more sense to remove the `Saved` interface and working with it in a generalised way, but thank you for picking up on this.

Reply:
thoughts ? : b974e868c1cef25abaddc7e91df392ba48a27bc9